ObjCVoiceCallKitQuickstart/ViewController.m

```diff
@@ -37,7 +37,8 @@ @implementation ViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     
-    [[TwilioVoice sharedInstance] setLogLevel:TVOLogLevelVerbose];
+    [TwilioVoice setLogLevel:TVOLogLevelVerbose];
+    [TwilioVoice setModule:TVOLogModulePJSIP logLevel:TVOLogLevelVerbose];
```

Comment:
I don't think we should set this to verbose by default in the quickstart. Seems quite noisy.